src/pyOpenMS/pxds/CubicSpline2d.pxd

```diff
@@ -10,6 +10,6 @@ cdef extern from "<OpenMS/MATH/MISC/CubicSpline2d.h>" namespace "OpenMS":
         CubicSpline2d(libcpp_vector[ double ] x, libcpp_vector[ double ] y) nogil except +
         CubicSpline2d(libcpp_map[ double, double ] m) nogil except +
 
-        double eval(double x) nogil except +
-        double derivatives(double x, unsigned order) nogil except +
+        double eval(double x) nogil except + # wrap-doc:Evaluates the spline at position x
+        double derivatives(double x, unsigned order) nogil except + # wrap-doc:Evaluates derivative of spline at position x
```

Comment:
What does order do?

Reply:
Returns first, second or third derivative of cubic spline.